.github/workflows/e2e-tests-manual.yaml

```
@@ -30,14 +59,17 @@ jobs:
         - 'dps-symmetric-key'
         - 'dps-x509'
 
+      max-parallel: 10
+
     steps:
     - uses: 'actions/checkout@v1'
       with:
         ref: "${{ github.events.input.branch }}"
```

```suggestion
        ref: "${{ github.event.inputs.branch }}"
```

https://github.com/Azure/iot-identity-service/commit/b59977078be65a4cc5d981cc036ced13f69b5f18#diff-1659b614bde0bc41517180808c08167a818ef7fc68e0f12ee672dc5bc0326531